pkg/postgres/schema/auth_machine_to_machine_configs.go

```diff
@@ -0,0 +1,67 @@
+// Code generated by pg-bindings generator. DO NOT EDIT.
+
+package schema
+
+import (
+	"fmt"
+	"reflect"
+
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/postgres"
+	"github.com/stackrox/rox/pkg/postgres/walker"
+	"github.com/stackrox/rox/pkg/sac/resources"
+)
+
+var (
+	// CreateTableAuthMachineToMachineConfigsStmt holds the create statement for table `auth_machine_to_machine_configs`.
+	CreateTableAuthMachineToMachineConfigsStmt = &postgres.CreateStmts{
+		GormModel: (*AuthMachineToMachineConfigs)(nil),
+		Children: []*postgres.CreateStmts{
+			&postgres.CreateStmts{
+				GormModel: (*AuthMachineToMachineConfigsMappings)(nil),
+				Children:  []*postgres.CreateStmts{},
+			},
+		},
+	}
+
+	// AuthMachineToMachineConfigsSchema is the go schema for table `auth_machine_to_machine_configs`.
+	AuthMachineToMachineConfigsSchema = func() *walker.Schema {
+		schema := GetSchemaForTable("auth_machine_to_machine_configs")
+		if schema != nil {
+			return schema
+		}
+		schema = walker.Walk(reflect.TypeOf((*storage.AuthMachineToMachineConfig)(nil)), "auth_machine_to_machine_configs")
+		referencedSchemas := map[string]*walker.Schema{
+			"storage.Role": RolesSchema,
+		}
+
+		schema.ResolveReferences(func(messageTypeName string) *walker.Schema {
+			return referencedSchemas[fmt.Sprintf("storage.%s", messageTypeName)]
+		})
+		schema.ScopingResource = resources.Access
+		RegisterTable(schema, CreateTableAuthMachineToMachineConfigsStmt)
+		return schema
+	}()
+)
+
+const (
+	// AuthMachineToMachineConfigsTableName specifies the name of the table in postgres.
+	AuthMachineToMachineConfigsTableName = "auth_machine_to_machine_configs"
+	// AuthMachineToMachineConfigsMappingsTableName specifies the name of the table in postgres.
+	AuthMachineToMachineConfigsMappingsTableName = "auth_machine_to_machine_configs_mappings"
```

Comment:
I just realized: this table exists just for the purpose of checking foreign key constraint to roles and there's no other use for it, correct?

Reply:
Yeah, seems like it.

Reply:
Yes that would be the sole purpose of that table since we can have multiple roles and are using a foreign key.